app/components/node-navbar/template.hbs

```diff
@@ -14,23 +14,26 @@
             </div>
             {{#bs-collapse collapsed=collapsedNav class='navbar-collapse'}}
                 <ul class='nav navbar-nav'>
-                    {{#node-navbar/link id=node.id useLinkTo=false extraClasses='project-title'}}
+                    {{#node-navbar/link nodeId=node.id useLinkTo=false extraClasses='project-title'}}
                         {{node.title}}
                     {{/node-navbar/link}}
-                    {{node-navbar/link id=node.id useLinkTo=false destination='files'}}
-                    {{node-navbar/link id=node.id useLinkTo=false destination='wiki'}}
-                    {{node-navbar/link id=node.id useLinkTo=false destination='analytics'}}
-                    {{node-navbar/link id=node.id useLinkTo=false destination='registrations'}}
+                    {{node-navbar/link nodeId=node.id useLinkTo=false destination='files'}}
+                    {{node-navbar/link nodeId=node.id useLinkTo=false destination='wiki'}}
+                    {{node-navbar/link nodeId=node.id useLinkTo=false destination='analytics'}}
+                    {{node-navbar/link nodeId=node.id useLinkTo=false destination='registrations'}}
 
                     {{#if node.currentUserIsContributor}}
-                        {{node-navbar/link id=node.id useLinkTo=false destination='contributors'}}
+                        {{node-navbar/link nodeId=node.id useLinkTo=false destination='contributors'}}
                         {{#if currentUserCanEdit}}
-                            {{node-navbar/link id=node.id useLinkTo=false destination='addons'}}
+                            {{node-navbar/link nodeId=node.id useLinkTo=false destination='addons'}}
                         {{/if}}
-                        {{node-navbar/link id=node.id useLinkTo=false destination='settings'}}
```

Comment:
Oh, so this was the cause of that weird "id already in use" error? Gonna have to remember never to use `id`...